pkg/operator/configobservation/scheduler/observe_scheduler.go

```diff
@@ -0,0 +1,83 @@
+package scheduler
+
+import (
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog"
+
+	"github.com/openshift/cluster-kube-scheduler-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-kube-scheduler-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
+)
+
+// observeSchedulerConfig lists the scheduler configuration and updates the name of the configmap that we want scheduler
+// to use as policy config
+func ObserveSchedulerConfig(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+	listers := genericListers.(configobservation.Listers)
+	errs := []error{}
+	prevObservedConfig := map[string]interface{}{}
+	policyConfigMapRootPath := []string{"algorithmSource", "policy", "configMap"}
+	policyConfigMapNamePath := append(policyConfigMapRootPath, "name")
+	policyConfigMapNamespacePath := append(policyConfigMapRootPath, "namespace")
```

Comment:
I think this value should always be pinned.  I'm pretty sure that's what I'm going to read below, but please comment the intent

Reply:
Sure. I will :)